client/src/locales/en-US.json

```diff
@@ -573,10 +573,12 @@
             "browse": "Go to location",
             "failedDetails": "Please make sure you are online."
         },
-        "uploadsLeft": "Importing {amount} items",
-        "copiesLeft": "Copying {amount} items",
-        "movesLeft": "Moving {amount} items",
-        "operationsLeft": "{amount} operations left"
+        "uploadsLeft": "{amount} imports pending",
+        "copiesLeft": "{amount} copies pending",
+        "movesLeft": "{amount} moves pending",
+        "itemsDoneAmount": "{amount} more operations done",
```

Comment:
Should use the plural for this.
`One more operation done | {amount} more operations done`
and use it with
`{ key: 'xxx.itemsDoneAmount', data: { amount: amount }, count: amount }`